addons/google_calendar/models/google_calendar.py

```diff
@@ -857,162 +834,92 @@ def update_events(self, cr, uid, lastSync=False, context=None):
                             else:
                                 raise e
```

Comment:
can't we replace the code in except block with below?

-
                        except Exception, e:
                            if e.code in (401, 410,):
                                pass
                            else:
                                raise e

-